plugins/pug.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: pug               # plugin name must match your manifest file name (e.g. foo.yaml)
+spec:
+  shortDescription: >-    # short description gets truncated at ~50 chars
+    Checks existing deprecated objects in a cluster 
+  # (optional) url of the project homepage
+  homepage: https://github.com/rikatz/kubepug
+  # (optional) use caveats field to show post-installation recommendations
+  caveats: |
+    * Kubepug downloads Kubernetes swagger.json to check objects and which are deprecated  
```

Comment:
These aren’t really caveats though. Maybe just delete.

Reply:
I think the part that talks about "master" default and that one should set the k8s version is good for the caveats section.

I presume, you also need to be allowed to list all objects? This could also be mentioned here.

Reply:
Yes, the thing is that the behavior might change from the default flags to an specific flag.

As an example, if I'm upgrading from k8s 1.15 to 1.17 and verify against master (and not against v1.17.0) it produces different results in NetworkPolicy alert (extensions/v1beta1 still exists in 1.17.0 but is already removed in master branch).

And yes, it needs a permission allowing to get all objects in the cluster so it will be mentioned! Thank you!!